app/web/src/features/dashboard/CommunityBrowser.tsx

```diff
@@ -0,0 +1,196 @@
+import {
+  Divider,
+  Link as MuiLink,
+  List,
+  ListItem,
+  ListItemText,
+} from "@material-ui/core";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import CircularProgress from "components/CircularProgress";
+import {
+  useCommunity,
+  useListSubCommunities,
+} from "features/communities/hooks";
+import { LOAD_MORE } from "features/dashboard/constants";
+import { Community } from "proto/communities_pb";
+import { forwardRef, ReactNode, Ref, useEffect, useRef, useState } from "react";
+import { Link } from "react-router-dom";
+import { routeToCommunity } from "routes";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "row",
+    "& > * + *": {
+      marginInlineStart: theme.spacing(2),
+    },
+  },
+  list: {
+    minWidth: "10rem",
+  },
+  loader: {
+    margin: theme.spacing("auto", 2),
+  },
+  selected: {
+    fontWeight: "bold",
+  },
+}));
+
+export default function CommunityBrowser() {
+  const classes = useStyles();
+  const [selected, setSelected] = useState<Community.AsObject[]>([]);
+
+  const globalCommunityQuery = useCommunity(1);
+
+  //react-query doesn't have useInfiniteQueries
+  //as a workaround, cache query results
+  //and only show "Load more" for last column
+  const query = useListSubCommunities(
+    selected?.[selected.length - 1]?.communityId || 1
+  );
+  const [cachedQueryResults, setCachedQueryResults] = useState<
+    {
+      data: Community.AsObject[];
+      hasMore?: boolean;
+    }[]
+  >([]);
+
+  const lastColumnRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    setTimeout(
+      () =>
+        lastColumnRef.current?.scrollIntoView({
+          behavior: "smooth",
+          block: "end",
+        }),
```

Comment:
Do we always want to scroll to the bottom of a long list of communities whenever selection changes? Might be a bit annoying if you unselect a previous selected community (maybe by accident), then re-select only to find it scrolls you all the way down when the community you want is near the top half of the list.

Reply:
This is for sideways scrolling, not vertical

Reply:
Don't you want `inline: "end"` instead? `block` scrolls things vertically according to MDN